### DISCUSSION TITLE
Test result issue [ Test result: ok. 0 passed ]

### DISCUSSION THREAD

**Discussion by LareineHan** [Cyfrin/foundry-full-course-cu#921] [Q&A] [answered]
So, I run the test for the test function below. 

```solidity
    function checkUpkeepReturnsFalseIfIthasNoBalance() public {
        // Arrange
        vm.warp(block.timestamp + interval + 1);
        vm.roll(block.number + 1);

        // Act
        (bool upkeepNeeded, ) = raffle.checkUpkeep("");

        // Assert
        assert(!upkeepNeeded); // which is true
    }
```

I used command
```zsh
forge test --mt checkUpkeepReturnsFalseIfIthasNoBalance
```

and it shows 
```zsh
Test result: ok. 0 passed; 0 failed; 0 skipped; finished in 5.03ms
 
Ran 1 test suites: 0 tests passed, 0 failed, 0 skipped (0 total tests)
```

What does it mean? When I see Patrick run this test on the video, his test result was '1 passed '. How can I fix it? Thanks for helping!

**Comment by theirrationalone** [comment, marked as answer]
Please prepend test function name with test. 

For e.g. testCheckupkeep() { // statements }

It's an indication to foundry that following are the functions for testing if their name start with word test. 

;`)

**Reply by LareineHan** [reply]
ah! Thank you. It works now.
